react/features/settings/components/web/SettingsDialog.js

```diff
@@ -126,14 +126,13 @@ class SettingsDialog extends Component<Props> {
  */
 function _mapStateToProps(state) {
     const configuredTabs = interfaceConfig.SETTINGS_SECTIONS || [];
```

Comment:
Food for thought: maybe we should drop the `SETTINGS_SECTIONS` thing altogether.